prod-aws/finance/account-enrichment.tf

```diff
@@ -0,0 +1,23 @@
+# tflint-ignore: terraform_naming_convention
+resource "kafka_topic" "account-enrichment-events" {
+  name               = "account-enrichment.events"
+  replication_factor = 3
+  partitions         = 10
+  config = {
+    "retention.bytes" = "-1"
+    "retention.ms"    = "-1" #forever
+    "cleanup.policy"  = "delete"
+  }
+}
+
+# tflint-ignore: terraform_naming_convention
```

Comment:
Is there a specific reason why we are unable to name the resource appropriately?"Is there a reason why we cannot name the resource properly?

Reply:
Hi Dani, I was using previous examples but I am unsure why we are suppressing the linter and not naming is appropriately. Any advice welcome ... :)

Reply:
@victoriaUW can you remove the `tflint` annotations?